.gitlab-ci.yml

```diff
@@ -50,12 +51,19 @@ G45J-MIRRORED-POOL:
     - chmod 775 ./Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/37Y6-create-raidz1-pool/create-raidz1-pool
     - ./Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/37Y6-create-raidz1-pool/create-raidz1-pool
 
-9ZWW-RAIDZ2-POOL:
+# 9ZWW-RAIDZ2-POOL:
+#   image: openebs/openshift-ci:latest
+#   stage: OPENEBS-SETUP
+#   script:
+#     - chmod 775 ./Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/9ZWW-create-raidz2-pool/create-raidz2-pool
+#     - ./Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/9ZWW-create-raidz2-pool/create-raidz2-pool
+
+KUSG-CSPC_STRIPE_POOL:
```

Comment:
@w3aman can you change the job name into `KUSG-CSPC-STRIPE-POOL`

Reply:
Done !